ui/src/components/PropertyDisplay.vue

```diff
@@ -23,7 +23,9 @@ export default class PropertyDisplay extends Vue {
   }
 
   get expanded (): string {
-    return this.createIdentifier(this.value)
+    return this.value.includes(':')
+      ? expand(this.value)
+      : this.createIdentifier(this.value)
```

Comment:
The underlying `Organization#createIdentifier` already does that but it treats strings a term template.

I would also pass the `Term` instead. (needs also update a type declarations)

```suggestion
    return this.createIdentifier(this.term)
```

Reply:
I don't see any call to `expand` in `createIdentifier`

Reply:
True, it does not expand. The input is either a Term (returned as-is) or a template, such as `cc:propertyTemplate` which gets concatenated. Prefixed URI is not an expected input

Reply:
Changed

Reply:
Crap, I need to change again then :D I wonder why it worked